### ISSUE TITLE
Label as recommend for closing

### ISSUE THREAD

**Issue by matt-dobson** [OHS-Hosting-Infrastructure/complaint-tracker#36]
User Story:
As a PS
I want to see that I've recommended closing a complaint
So that I can see that that my actions are complete

Acceptance:
Given that I am a PS
When I look at a complaint
Then I c an see that it has been recommended to close

**Comment by lauranash18f** [comment]
I believe this is covered by the status in the table.

**Comment by matt-dobson** [comment]
Yes. We should write a Status field story to cover this and other statuses. It is TBD in the current state.
